### PR TITLE
feat: update generated samples docs

### PR DESCRIPTION
**Pull request by aandreassa**
Requirements listed in b/240118237.

### CODE REVIEW

File: gapic-generator/lib/gapic/presenters/snippet_presenter.rb [hunk 1]
```diff
@@ -118,8 +118,11 @@ def snippet_name
 
       def description
         @config&.metadata&.snippet_description ||
-          "This is an auto-generated example demonstrating basic usage of #{client_type}##{method_name}. " \
-          "It may require modification in order to execute successfully."
+          "This snippet has been automatically generated and should be regarded as a code template only." \
```
Comment (dazuma) [inline gapic-generator/lib/gapic/presenters/snippet_presenter.rb:121]: The backslashes will only concatenate strings directly. You'll need to explicitly embed newlines (`\n`) in the string if you want them formatted on separate lines.

**Review by dazuma — CHANGES_REQUESTED** [review]
You'll also need to update the golden outputs in order to get the tests to pass.

File: gapic-generator/lib/gapic/presenters/snippet_presenter.rb (hunk 1 above)
Comment (aandreassa) [reply gapic-generator/lib/gapic/presenters/snippet_presenter.rb]: Thanks Daniel! The codegen wraps comments to a specific length & trims whitespace (both ends), so I added the non-ASCII space in order to keep the docs align (i.e. \u00A0). PLMK if you have a better method.

The rest done via`toys bin` + `toys gen`.

Comment (aandreassa) [reply gapic-generator/lib/gapic/presenters/snippet_presenter.rb]: The double `\n` approach did not work, but I updated the template file accordingly. It renders correct in HTML -- I will send you the image in chat.

**Review by dazuma — APPROVED** [review]
Great, that works well. Thanks!

**Pull request merged**